drinks/tasks/models.py

```diff
@@ -1,3 +1,27 @@
 from django.db import models
+from django.contrib.auth.models import User
+
 
 # Create your models here.
+class Ingredient(models.Model):
+    name = models.CharField(max_length=220)
+    quantity = models.CharField(max_length=50, blank=True, null=True)  # 400
```

Comment:
nie bardzo wiadomo co to jest to 400 i czemu Charfield

Reply:
poprawione